include/wlr/types/wlr_idle_inhibit_v1.h

```diff
@@ -0,0 +1,27 @@
+#ifndef WLR_TYPES_WLR_IDLE_INHIBIT_V1_H
+#define WLR_TYPES_WLR_IDLE_INHIBIT_V1_H
+
+#include <wayland-server.h>
+
+struct wlr_idle_inhibit_v1 {
```

Comment:
This represents a idle-inhibit-manager global, so it should be named `wlr_idle_inhibit_manager_v1`.